conf/messages

```diff
@@ -68,20 +68,22 @@ home.unique.taxpayer.reference                                  = Unique Taxpaye
 home.utr                                                        = UTR: {0}
 home.payments.heading                                           = Next payments due
 home.payments.no-payments-due                                   = No payments due
-home.payments.view                                              = View what you owe
+home.payments.view                                              = Check what you owe
 home.updates.heading                                            = Next updates due
-home.updates.view                                               = View next updates
+home.updates.view                                               = View update deadlines
+home.paymentHistoryAndCredit.heading                            = Payment history and credit
+home.paymentHistoryAndCredit.view                               = Payment and refund history
 home.overdue.date.payment.count                                 = {0} OVERDUE PAYMENTS
 home.overdue.date.update.count                                  = {0} OVERDUE UPDATES
 home.overdue.date                                               = OVERDUE
-home.tax-years.heading                                          = Tax years
-home.tax-years.view                                             = View tax years
+home.tax-years.heading                                          = Returns
+home.tax-years.view                                             = View all tax years
 home.tax-years.viewPayments                                     = View payment history
-home.manage-income-tax.heading                                  = Manage Income Tax
 home.your-returns.heading                                       = Manage Income Tax
 home.your-returns.description                                   = Use this service to report untaxed income and any tax relief you’re claiming
 home.your-returns.agent.description                             = Use this service to report untaxed income and any tax relief you’re claiming.
-home.your-returns.updatesLink                                   = Update and submit an Income Tax Return
+home.your-returns.updatesLink                                   = Update and submit your {0} to {1} return
+home.returns.viewLink                                           = View your current {0} to {1} return
 home.update-and-submit-your-returns.Link                        = Update and submit an Income Tax Return
```

Comment:
@andriikondiuk - I don't think this is being used anymore.

Reply:
Done, thanks